Headers/species_mod.F90

```diff
@@ -158,6 +164,25 @@ MODULE Species_Mod
      REAL(fp)           :: WD_RainoutEff(3) ! Temperature-dependent scale
                                             !  factors for rainout efficiency
 
+     ! TransportTracers parameters
+     CHARACTER(LEN=80)  :: Snk_Horiz        ! Where to apply sink horizontally?
+     REAL(fp)           :: Snk_LatMin       ! Minimum latitude for applying sink
+     REAL(fp)           :: Snk_LatMax       ! Maximum latitude for applying sink
+     CHARACTER(LEN=80)  :: Snk_Mode         ! Sink mode of tracer
+     REAL(fp)           :: Snk_Period       ! Sink period in days
+     REAL(fp)           :: Snk_Value        ! Sink value
+     CHARACTER(LEN=80)  :: Snk_Vert         ! Where to apply sink vertically?
+     LOGICAL            :: Src_Add          ! T- add values; F- replace values
+     CHARACTER(LEN=80)  :: Src_Horiz        ! Where to apply source horizontally
+     REAL(fp)           :: Src_LatMin       ! Minimum latitude for applying src
+     REAL(fp)           :: Src_LatMax       ! Maximum latitude for applying src
+     CHARACTER(LEN=80)  :: Src_Mode         ! Source mode of tracer
+     REAL(fp)           :: Src_PresMin      ! Minimum pressure for applying src
```

Comment:
What are the units for this?

Reply:
The units are now specified on RTD under [Transport tracer properties](https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/species-guide.html#transport-tracer-properties).